### PR TITLE
[improve][doc] Add code example for WebSocket API to use TLS with CLI tools

### PR DESCRIPTION
**Pull request by momo-jun**
### Modifications

1. Add code example for WebSocket API to use TLS with CLI tools
2. Link to WebSocket configurations about TLS

The preview looks good.
image

### Documentation
  
- [x] `doc`

### CODE REVIEW

**Comment by momo-jun** [comment]
Hi @BewareMyPower, can you pls help review this doc PR? Thanks.
//cc @Anonymitaet

File: site2/docs/security-tls-transport.md [hunk 1]
```diff
@@ -273,3 +273,49 @@ var client = PulsarClient.Builder()
...
+
+#### WebSocket API
+
+
+```python
+import websockets
+import asyncio
+import base64
+import json
+import ssl
```
Comment (BewareMyPower) [inline site2/docs/security-tls-transport.md:285]:
```suggestion
import ssl
import pathlib
```

File: site2/docs/security-tls-transport.md [hunk 2]
```diff
@@ -273,3 +273,49 @@ var client = PulsarClient.Builder()
...
+
+ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+client_pem = pathlib.PATH(__file__).with_name("pulsar_client.pem")
```
Comment (BewareMyPower) [inline site2/docs/security-tls-transport.md:288]:
```suggestion
client_pem = pathlib.Path(__file__).with_name("pulsar_client.pem")
```

**Review by BewareMyPower — CHANGES_REQUESTED** [review]

**Comment by momo-jun** [comment]
@BewareMyPower @tuteng PTAL.

File: site2/docs/security-tls-transport.md [hunk 3]
```diff
@@ -324,6 +324,46 @@ var client = PulsarClient.Builder()
...
+
+```python
+import websockets
+import asyncio
+import base64
+import json
+import ssl
+import pathlib
+
+ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
```
Comment (nodece) [inline site2/docs/security-tls-transport.md:338]:
```suggestion
ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
client_cert_pem = pathlib.Path(__file__).with_name("client.cert.pem")
client_key_pem = pathlib.Path(__file__).with_name("client.key.pem")
ca_cert_pem = pathlib.Path(__file__).with_name("ca.cert.pem")
ssl_context.load_cert_chain(certfile=client_cert_pem, keyfile=client_key_pem)
ssl_context.load_verify_locations(ca_cert_pem)
# websocket producer uri wss, not ws
uri = "wss://localhost:8080/ws/v2/producer/persistent/public/default/testtopic"
```

**Review by nodece — APPROVED** [review]

**Pull request merged**